packages/emulation/libretro-mupen64plus/package.mk

```diff
@@ -36,30 +36,33 @@ PKG_LIBPATH="$PKG_LIBNAME"
 PKG_LIBVAR="MUPEN64PLUS_LIB"
 
 make_target() {
-  case $PROJECT in
-    RPi)
-      case $DEVICE in
-        RPi)
-          make platform=rpi
-          ;;
-        RPi2)
-          make platform=rpi2
-          ;;
-      esac
-      ;;
-    imx6)
-      make platform=imx6
-      ;;
-    WeTek_Play|WeTek_Core|Odroid_C2|WeTek_Hub|WeTek_Play_2)
-      if [ "$TARGET_ARCH" = "aarch64" ]; then
-        make platform=aarch64
-      else
-        make platform=armv7-neon-gles-cortex-a9
-      fi
+
+  if target_has_feature neon; then
```

Comment:
@CGarces @Raybuntu has asked me to comment as this issue has come up again. this is the offending line causing the issue with repo-tool.

Because repo-tool does not source the function the command not found error is displayed and stops addons being built.

This can be resolved with adding . config/functions to repo-tool but I'm unsure if this is the appropriate fix, @MilhouseVH should be able to advise better on this issue.

Reply:
Hi @adamg88 the problem is that I haven't reproduced the issue months ago when I test the repo-tool.
Probably was my fault, I'll try again, but should be great it you give me some hint (one .sh script?) to reproduce the issue over the current master.

Reply:
I'm trying to reproduce it right now.

Reply:
`
cd tools/mkpkg
./update_retroplayer-addons
`

Reply:
OK yes I can reproduce it.

Reply:
The correct fix is for scripts to source packages using `config/options <pkg-name>`, rather than sourcing the `package.mk` directly which leads to the `command not found` (and other) errors. Not strictly the fault of this PR.

Reply:
I agree. We should not block this PR nor ask the author to fix the script. I didn't have much time but I tried that earlier today. It didn't work entirely because the script is a bit messy and I suggest to rework it in a seperate PR.